### PR TITLE
feat: remove null/blank from CMO (FPLA-1370)

### PR DESCRIPTION
**Pull request by pd-dudkiewicz**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-1370

### Change description ###

Removed blank placeholders from docmosis template.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by joez-f** [comment]
can you attach an example CMO PDF to the PR?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java [hunk 1]
```diff
@@ -177,14 +177,6 @@ private String getFirstApplicantName(CaseData caseData) {
             .collect(toList());
     }
 
-    private CaseManagementOrder getCaseManagementOrder(CaseData caseData) {
-        if (caseData.getCaseManagementOrder() != null) {
-            return caseData.getCaseManagementOrder();
-        }
-
-        return null;
-    }
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java]: What even is this 🤣

**Comment by pd-dudkiewicz** [comment]
[draft-case_management_order.pdf](https://github.com/hmcts/fpl-ccd-configuration/files/4375048/draft-case_management_order.pdf)

Attached file as requested by @joez-f

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java (hunk 1 above)
Comment (pd-dudkiewicz) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java]: 🤷‍♂

**Pull request merged**